app/controllers/registrations_controller.rb

```diff
@@ -36,21 +36,21 @@ def start
       password: params.dig(:user, :password),
       password_confirmation: params.dig(:user, :password),
       phone: params.dig(:user, :phone),
```

Comment:
Set `phone` to `nil` if not `request_auth_wants_mfa?`.  Currently this allows a user to supply a phone number, which we then save without confirming.

Reply:
gotcha